openedx/adg/lms/student/tests/test_helpers.py

```diff
@@ -0,0 +1,209 @@
+    """
+    user = UserFactory()
+    UserProfileFactory(user=user)
+    return user
+
+
+@pytest.mark.django_db
+def test_compose_and_send_adg_activation_email(mocker, user_with_profile):  # pylint: disable=redefined-outer-name
+    """Test 'compose_and_send_adg_activation_email' helper method of student"""
+
+    mock_send_email = mocker.patch.object(student_helpers, 'send_mandrill_email')
+
+    mock_get_configuration_value = mocker.patch.object(student_helpers.configuration_helpers, 'get_value')
+    mock_get_configuration_value.return_value = settings.LMS_ROOT_URL
+
+    activation_key = 'some_activation_key'
+    expected_context = {
+        'first_name': user_with_profile.profile.name.split()[0],
+        'activation_link': '{root_url}/activate/{activation_key}'.format(
+            root_url=mock_get_configuration_value.return_value,
+            activation_key=activation_key)
+    }
+
+    student_helpers.compose_and_send_adg_activation_email(user_with_profile, activation_key)
+    mock_send_email.assert_called_once_with(
+        MandrillClient.USER_ACCOUNT_ACTIVATION, user_with_profile.email, expected_context
+    )
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('fake_request', [RequestFactory().put("/dummy", secure=True),
+                                          RequestFactory().put("/dummy", secure=False)])
+def test_compose_and_send_adg_password_reset_email(
+    mocker, user_with_profile, fake_request  # pylint: disable=redefined-outer-name
+):
+    """Test 'compose_and_send_adg_password_reset_email' helper method of student"""
+
+    mocker.patch.object(student_helpers, 'int_to_base36')
+    mocker.patch.object(student_helpers, 'default_token_generator')
+    mock_reverse = mocker.patch.object(student_helpers, 'reverse')
+    mock_send_email = mocker.patch.object(student_helpers, 'send_mandrill_email')
+    mock_get_configuration_value = mocker.patch.object(student_helpers.configuration_helpers, 'get_value')
+
+    mock_get_configuration_value.return_value = settings.SITE_NAME
+    mock_reverse.return_value = 'password_reset_confirm'
+
+    expected_context = {
+        'first_name': user_with_profile.profile.name.split()[0],
+        'reset_link': '{protocol}://{site}{link}?track=pwreset'.format(
+            protocol='https' if fake_request.is_secure() else 'http',
+            site=mock_get_configuration_value.return_value,
+            link=mock_reverse.return_value
+        )
+    }
+
+    student_helpers.compose_and_send_adg_password_reset_email(user_with_profile, fake_request)
+    mock_send_email.assert_called_once_with(MandrillClient.PASSWORD_RESET, user_with_profile.email, expected_context)
+
+
+@pytest.mark.django_db
+def test_compose_and_send_adg_update_email_verification(
+    mocker, user_with_profile  # pylint: disable=redefined-outer-name
+):
+    """Test 'compose_and_send_adg_update_email_verification' helper method of student"""
```

Comment:
Please use edx recommended docstring style, in all tests

"""
Here
"""

In docstring, for code instead of using ' use `